managers/oclcClassify.py

```diff
@@ -124,7 +125,10 @@ def execQuery(self):
             [str] -- A string of XML data comprising of the body of the
             Classify response.
         """
-        classifyResp = requests.get(self.query, timeout=10)
+        classifyResp = requests.get(self.query, \
+            headers={'X-OCLC-API-Key': os.environ['OCLC_API_KEY']}, \
```

Comment:
also, you *shouldn't* need these line break characters within a function call (unless I'm missing something!)